scm_v3c/radio.c

```diff
@@ -158,7 +158,8 @@ void receive_packet(bool timeout) {
 // If timeout is set false, then the function may block indefinitely
 void receive_packet_length(uint8_t pkt_len, bool timeout) {
     radio_vars.radio_mode = RX_MODE;
-    radio_vars.rxPacket_len = pkt_len;
+    radio_vars.rxPacket_len =
+        pkt_len <= RX_PKT_ANY_LEN ? pkt_len : RX_PKT_ANY_LEN;
```

Comment:
Do we have min/max macros? Probably more readable to use min(pkt_len, RX_PKT_ANY_LEN)

Reply:
I don't think so, I'll add it in a separate PR